.github/workflows/release-latest.yml

```diff
@@ -28,6 +28,22 @@ jobs:
       - run: |
           git tag v${{ fromJson(steps.pkg.outputs.json).version }}
           git push origin v${{ fromJson(steps.pkg.outputs.json).version }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6 # 2.0.0
+        with:
+          install: true
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
```

Comment:
Why limit parallelism here?

Reply:
This is copypasta from the Java version of this.

https://github.com/DataDog/dd-trace-java/blob/1e9abac39fbc42906f1cb107cdf64e47074d692c/.github/workflows/test-lib-injection.yaml#L45-L51